includes/class.llms.cache.helper.php

```diff
@@ -106,17 +108,75 @@ public function maybe_no_cache() {
 			)
 		);
 
-		if ( is_page( $ids ) ) {
+		/**
+		 * Filter whether or not LifterLMS will send nocache headers.
+		 *
+		 * @since [version]
+		 *
+		 * @param bool $no_cache Whether or not LifterLMS will send nocache headers.
+		 */
+		$do_not_cache = apply_filters( 'llms_no_cache', is_page( $ids ) || is_quiz() );
+
+		if ( $do_not_cache ) {
+
+			add_filter( 'nocache_headers', array( __CLASS__, 'additional_nocache_headers' ), 99 );
 
 			llms_maybe_define_constant( 'DONOTCACHEPAGE', true );
 			llms_maybe_define_constant( 'DONOTCACHEOBJECT', true );
 			llms_maybe_define_constant( 'DONOTCACHEDB', true );
 			nocache_headers();
 
+			remove_filter( 'nocache_headers', array( __CLASS__, 'additional_nocache_headers' ), 99 );
+
 		}
 
 	}
 
+	/**
+	 * Set additional nocache headers.
+	 *
+	 * @since [version]
+	 *
+	 * @see wp_get_nocache_headers()
+	 *
+	 * @param array $headers {
+	 *     Header names and field values.
+	 *
+	 *     @type string $Expires       Expires header.
+	 *     @type string $Cache-Control Cache-Control header.
+	 * }
+	 * @return array
+	 */
+	public static function additional_nocache_headers( $headers ) {
+
+		// First tree are the default ones.
+		$nocache_headers = array(
+			'no-cache',
+			'must-revalidate',
+			'max-age=0',
+			'no-store',
+		);
+
+		$headers['Cache-Control'] = implode(
```

Comment:
@pondermatic sorry :P

Reply:
Nope. I veto this unreadable piece of golf.

Reply:
I veto this with a smile and love.

Reply:
fixed in 2fcd31b